Bottom: d7dee332fb825a63adf8e5f9850bf89dc6b86fcc
Top:    34129bfd8b8c50b31d51d57fd15bb3602ba0d80e
Author: Jason Wang <jasowang@redhat.com>
Date:   2016-05-23 18:44:41 +0800

vhost: switch to use bit 63

Signed-off-by: Jason Wang <jasowang@redhat.com>


---

diff --git a/drivers/vhost/net.c b/drivers/vhost/net.c
index 20be32f..6961630 100644
--- a/drivers/vhost/net.c
+++ b/drivers/vhost/net.c
@@ -1096,7 +1096,7 @@ static int vhost_net_set_features(struct vhost_net *n, u64 features)
 		mutex_unlock(&n->dev.mutex);
 		return -EFAULT;
 	}
-	if ((features & (1 << VHOST_F_DEVICE_IOTLB))) {
+	if ((features & (1ULL << VHOST_F_DEVICE_IOTLB))) {
 		if (vhost_init_device_iotlb(&n->dev, true))
 			return -EFAULT;
 	}
diff --git a/include/uapi/linux/vhost.h b/include/uapi/linux/vhost.h
index bb90660..19493ee 100644
--- a/include/uapi/linux/vhost.h
+++ b/include/uapi/linux/vhost.h
@@ -171,7 +171,7 @@ struct vhost_memory {
 /* vhost-net should add virtio_net_hdr for RX, and strip for TX packets. */
 #define VHOST_NET_F_VIRTIO_NET_HDR 27
 /* Vhost have device IOTLB */
-#define VHOST_F_DEVICE_IOTLB 28
+#define VHOST_F_DEVICE_IOTLB 63
 
 /* VHOST_SCSI specific definitions */
